README.md

```diff
@@ -0,0 +1,52 @@
+#
+
+![logo](https://raw.githubusercontent.com/pybamm-team/pybamm-tea/main/docs/pybamm_tea_logo.png)
+
+<div align="center">
+
+[![Documentation Status]]
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/pybamm-team/pybamm-tea/blob/main/)
+
+</div>
+
+# PyBaMM-TEA
+
+This repository contains the work of the "Google Summer of Code" project on a techno-economic analysis library for battery cells, which can be combined with PyBaMM's functionality.
+So far, there is a method to visualize mass- and volume loadings of an electrode stack and to estimate energy densities without simulation. The project further aims to estimate cell metrics with simulations (e.g. a Ragone plot) and manufacturing metrics with a Process-based Cost Model.
+
+
+## Installation
+
```

Comment:
can you add instructions on how to clone the repo here

Reply:
Can you add instructions for mac/linux/windows, e.g.


### Linux/Mac OS
1. Clone the repository using git
```bash
git clone https://github.com/pybamm-team/PyBaMM-TEA.git
```
or download (and extract) the ZIP file by clicking the green button on the top right of the page.
2. Change into the `pybamm-tea` directory 
```bash
cd pybamm-tea
```
3. Create a virtual environment
```bash
virtualenv env
```
4. Activate the virtual environment 
```bash
source env/bin/activate
```
5. Install the package
```bash 
pip install .
```
To install the project in editable mode use `pip install -e .`.

### Windows
To install the package from the local project path on Windows use the following commands:

1. Clone the repository using git
```bash
git clone https://github.com/pybamm-team/PyBaMM-TEA.git
```
or download (and extract) the ZIP file by clicking the green button on the top right of the page.
2. Change into the `pybamm-tea` directory 
```bash
cd pybamm-tea
```
3. Create a virtual environment
```bash
virtualenv env
```
4. Activate the virtual environment 
```bash
\path\to\env\Scripts\activate
```
where `\path\to\env` is the path to the environment created in step 3 (e.g. `C:\Users\'Username'\env\Scripts\activate.bat`).
5. Install the package
```bash 
pip install .
```
To install the project in editable mode use `pip install -e .`.

As an alternative, you can set up [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about). This allows you to run a full Linux distribution within Windows.